docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -55,6 +55,26 @@ There you can find links to upgrade notes for other versions too.
         ```
 - update your tests to use interfaces of factories fetched from dependency injection container
     -  update tests same way as in PR ([#970](https://github.com/shopsys/shopsys/pull/970/files))
+- check your VAT calculations after it was modified in `shopsys/framework` ([#1129](https://github.com/shopsys/shopsys/pull/1129))
+    - see [the description of the PR](https://github.com/shopsys/shopsys/pull/1129) to understand the scope of this change
```

Comment:
```suggestion
    - we strongly recommend seeing [the description of the PR](https://github.com/shopsys/shopsys/pull/1129) to understand the scope of this change
```

Reply:
We offer the link to PR in every upgrade instruction. I would like to highlight it here because they really should see it.